tower/src/balance/p2c/service.rs

```diff
@@ -78,6 +78,7 @@ enum Error<E> {
     Canceled,
 }
 
+#[allow(clippy::len_without_is_empty)]
```

Comment:
should we just consider adding an `is_empty` method?

Reply:
Sure. I just wasn't sure if it was appropriate here. Not very familiar with this API. Added `is_empty` in 58821d7826a07